examples/hello_world.go

```diff
@@ -103,15 +103,15 @@ func Check(e webui.Event) string {
 func main() {
 
 	// New window
-	var my_window = webui.NewWindow()
```

Comment:
Using a short scoped variable declarations as an approach that is closer to Gos spirit.